src/Hl7.Fhir.Core/Serialization/FhirXmlParser.cs

```diff
@@ -30,7 +30,7 @@ public FhirXmlParser(ParserSettings settings=null) : base(settings)
                     DisallowSchemaLocation = Settings.DisallowXsiAttributesOnRoot,
                 };
 
-        public Base Parse(string xml, Type dataType)
+        public Base Parse(string xml, Type dataType = null)
```

Comment:
There is another overload on Parse() which takes an XmlReader, and is also missing the `= null`.  The same is true for these functions on the FhirJsonParser.cs